tic_tac_toe/game.py

```diff
@@ -24,16 +29,26 @@ def _position_is_valid(position):
 
     Returns True if given position is valid, False otherwise.
     """
-    pass
-
+    try: 
+        row = position[0]
+        column = position[1]
+        rowIsInRange = (0 <= row <= 2)
+        columnIsInRange = (0 <= column <= 2)
+        tupleIsLengthTwo = (len(position) == 2)
+        return (rowIsInRange and columnIsInRange and tupleIsLengthTwo)
+    except:
```

Comment:
This try/except pattern is a nice concise way to avoid writing if statements for multiple lines that may result in an error. I will remember this :)